inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/heartbeat/HeartbeatManager.java

```diff
@@ -168,4 +174,11 @@ private HeartbeatMsg buildHeartbeat() {
         heartbeatMsg.setStreamHeartbeats(streamHeartbeats);
         return heartbeatMsg;
     }
+
+    private boolean validReportInfo(SourceReportInfo reportInfo) {
+        if (StringUtils.isBlank(reportInfo.getIp()) || StringUtils.isBlank(reportInfo.getPort())) {
```

Comment:
Can simply to `return StringUtils.isNotBlank(reportInfo.getIp()) && StringUtils.isNotBlank(reportInfo.getPort());`.

Reply:
done